scripts/comparator/setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
Hm, could we just call into the docker-integration-tests setup.sh with `SERVICES=(m3comparator m3query)`?

Reply:
Mostly yeah, but the dockerfiles are different; would prefer to avoid more bash hacking for now but can probably make it work if you're keen on it

Reply:
Hm, it's fine.  We should clean up some of these build tools at some point though, spreading more bash in a non-DRY way will get painful eventually.